doc/whats-new.rst

```diff
@@ -65,6 +65,9 @@ Bug fixes
   By `András Gunyhó <https://github.com/mgunyho>`_.
 - Avoid use of random numbers in `test_weighted.test_weighted_operations_nonequal_coords` (:issue:`6504`, :pull:`6961`).
   By `Luke Conibear <https://github.com/lukeconibear>`_.
+- Use ``keep_attrs=True`` in grouping and resampling operations by default (:issue:`7012`).
```

Comment:
I think they were kept without flox - do you want to mention this?

Reply:
Dataset level attrs were lost earlier: https://github.com/pydata/xarray/blob/b018442c8dfa3e71ec35e294de69e2011949afec/xarray/core/dataset.py#L5603

Now its always preserved whether you use flox or not. Should I clarify the second line here more.

Reply:
> Should I clarify the second line here more.

Thanks for the clarification - fine to leave as is for me.